.gitlab-ci.yml

```diff
@@ -0,0 +1,126 @@
+image: "ablock/dash-gitlabci-builder"
```

Comment:
Would prefer some base image here instead tbh even if it's going to be a bit slower.

Reply:
Changed to use `ubuntu:bionic` and implemented apt caching to speed things up again